components/Cmip6MonthlyChartControls.vue

```diff
@@ -67,6 +67,39 @@ chartStore.labels = {
   },
 }
 
+// Sea ice concentration data is only available for some models.
+if (props.datasetKeys?.includes('siconc')) {
```

Comment:
Tricky.  Again I'd prefer that this be a prop that is more about the functionality of this component and doesn't bind this component to need to "know" about specific dataset names.  But I don't think it's obvious what or how to do that right now.  Could you make a ticket for this?

Reply:
I've made a ticket referencing this #198